### ISSUE TITLE
align by first assignment operator (= :) each line

### ISSUE THREAD

**Issue by shawn-yee** [OldStarchy/Align-Spaces#18]
I think only the first =  or : alignment is enough for each line

For example, the alignment of arrows or other operators will cause too many meaningless whitespace, because if variable names are very long, the whitespace will increase exponentially,there are many such examples.

In short, one is enough.

```javascript
// expected
const f1      = (x) => x + 1
const f111111 = (x, y, z) => x + y + z

```javascript
// now
const f1      = (x)       => x + 1
const f111111 = (x, y, z) => x + y + z

**Comment by OldStarchy** [comment]
The example you provided is not accurate, the grouping system works by checking each line contains the same operators in the same order (up to the length of the shortest line).

In your example, the operators are

```
=  =  +
=  ,  ,  =  +  +
   ^  ^
   \--\-- different
```

The first line only contains 3 operators, so only the first 3 operators on the second line are compared, and since the `=` is an "assignment" operator and the `,` is a comma, the lines aren't grouped.

In order to get an example similar to yours I had to remove the commas

```javascript
const f1 = (x) => x + 1
const f111111 = (x y z) => x + y + z

//becomes

const f1      = (x)     => x + 1
const f111111 = (x y z) => x + y + z
```

```
=  =  +
=  =  +  +
^  ^  ^
\--\--\-- same
```

Anyway, I'll add a setting that will prevent aligning operators after the first assignment.

**Comment by OldStarchy** [comment]
New config `"align-spaces.skip-after-first-assignment": true` available in version 0.4.0.

**Comment by shawn-yee** [comment]
Yes, I found something wrong with the example I gave
I have tried the new version, 0.4.0
So far, no problem, thanks

```javascript
//expected
const rrrrrrr = 11
const g       = 22
const b       = 33
const x1      = rrrrrrr / 255
const x2      = g / 255
const x3      = b / 255

//before
const rrrrrrr = 11
const g       = 22
const b       = 33
const x1      = rrrrrrr / 255
const x2      = g       / 255
const x3      = b       / 255
